docs/Generator.html

```diff
@@ -429,9 +431,41 @@
 </tr>
 </table>
 
+
 <p>Of further note is the existence of two sets of the IntGen classes, representing the individual operations. There are the cProtIntGen class descendants, which are used for prototyping the connections between the operations - it's easy to just chain several operations after each other and they automatically use the correct array dimensions. However, it is possible to further optimize the calculations by moving the array dimensions into template parameters (so that they are, in fact, constant from the code's point of view, and so highly optimizable). This is what the cIntGen class descendants do. Unfortunately, this optimization makes it difficult to change the operation chain - when a new operation is added or removed in the chain, the array sizes for the rest of the chain change and they all have to be updated manually. So the optimal strategy was to use the cProtIntGen classes to find out the best-looking combination of operations, and once the combination was found, to rewrite it using cIntGen classes for performance.
 </p>
 
+<br />
+
+Here is a visualizer where you can play with the algorithm that the Grown biome generator uses:
+<table>
+	<tr>
+		<td rowspan="5">
+			<canvas id="stage" width="400" height="400"></canvas>
```

Comment:
How about I give a warning that rendering larger than 600x600 can lag the browser?